src/IFrameNavigator.ts

```diff
@@ -1442,9 +1454,19 @@ export default class IFrameNavigator implements Navigator {
   }
 
   private navigate(readingPosition: ReadingPosition): void {
+    console.log("readingPosition", readingPosition);
     this.hideIframeContents();
     this.showLoadingMessageAfterDelay();
     this.newPosition = readingPosition;
+    if(this.encryption) {
+      let isEncrypted = this.encryption.resources.find((resource: string) => {
+        return readingPosition.resource.includes(resource);
```

Comment:
In the sample that I was using, resources were of the address `opf/chapter1_part1.xhtml` whereas readingPositions were of the address `www.host.com/blah/dee/blah/opf/chapter1_part1.xhtml`

Do we know if this will hold for everything?

Reply:
I don't know; what is a readingPosition in this context? Like which chapter you're currently reading?

Reply:
Yeah, the link to the chapter.  `readingPosition.resource` is the direct link to the chapter xhtml.